### PR TITLE
Ok1ez - MDT Edit with new Map

### PR DESCRIPTION
**Pull request by OK1ez**

### CODE REVIEW

**Comment by OK1ez** [comment]
Skjermbilde_20230206_165001

File: ui/app.js [hunk 1]
```diff
@@ -669,7 +669,7 @@ $(document).ready(() => {
     ".manage-incidents-create",
     function () {
       let tempalte =
-        "📝 Summary:\n\n[Insert Report Summary Here]\n\n🧍 Hostage: [Name Here]\n\n🔪 Weapons/Items Confiscated:\n\n· [Insert List Here]\n\n-----\n💸 Fine:\n⌚ Sentence:\n-----";
+        "Summary:\n\n[Insert Report Summary Here]";
```
Comment (MonkeyWhisper) [inline ui/app.js]: Please re-add this, since we are merging into the main branch, we want to keep all original things and just leave the current edit

**Review by MonkeyWhisper — CHANGES_REQUESTED** [review]
Great job, please see change requested

**Review by MonkeyWhisper — APPROVED** [review]
I tested this myself and everything seems to be working normal, any issues, please report it.

**Pull request merged**